input/UDP_connect.py

```diff
@@ -0,0 +1,88 @@
+"""Phone and UDP socket."""
+import socket
+import numpy as np
+
+
+class UdpSocket:
+    """An udp socket."""
+
+    def __init__(self, ip_address: str = "0.0.0.0", port: int = 2055,
```

Comment:
We can use `ipaddress.ip_address` as a datatype here for encapsulation, validation and better type annotations.